### PR TITLE
Some more icons and fixes

### PR DESCRIPTION
**Pull request by Atharv-Pathak-14**
Icon added :-
Arch linux
Red hat dark 
Red hat light
Nodejs main logo light 
Node js main logo dark

Fixes :-
Removed the fake java icon and added the legit one 

The colours of the icon which was used before differ from the real logo 

Contact - Atharv Pathak#0001

### CODE REVIEW

File: src/icons.ts [hunk 1]
```diff
@@ -31,6 +31,7 @@ export const icons: Icon[] = [
   i("Apple", { [d]: "apple-dark.svg", [l]: "apple-light.svg" }),
   i("AWS", { [c]: "aws.svg" }),
    i("Azure", { [c]: "azure.svg" }),
+   i("ArchLinux", { [c]: "arch.svg" }),
```
Comment (FarazPatankar) [inline src/icons.ts:34]: Indentation issue on Azure and ArchLinux.

File: src/icons.ts [hunk 2]
```diff
@@ -104,7 +105,7 @@ export const icons: Icon[] = [
   i("n8n", { [c]: "n8n.svg" }),
   i("NextJS", { [c]: "nestjs.svg" }),
   i("NocoDB", { [c]: "nocodb.png" }),
-  i("NodeJS", { [c]: "nodejs.svg" }),
+  i("NodeJS", { [c]: "nodejs.svg", [d]: "nodejs-black.svg", [l]: "nodejs-white.svg" }),
```
Comment (FarazPatankar) [inline src/icons.ts:108]: Can we change the naming to `dark` and `light` instead of `black` and `white` for consistency?

File: src/icons.ts [hunk 3]
```diff
@@ -116,6 +117,7 @@ export const icons: Icon[] = [
   i("Rails", { [c]: "rails.svg" }),
   i("Railway", { [d]: "railway-dark.svg", [l]: "railway-light.svg" }),
   i("React", { [c]: "react.svg" }),
+   i("RedHat", { [d]: "Red-hat-b.svg", [l]: "Red-hat-w.svg" }),
```
Comment (FarazPatankar) [inline src/icons.ts:120]: Same naming suggestion here.

**Review by FarazPatankar — CHANGES_REQUESTED** [review]
Thanks for the PR, left some comments for minor changes! :)

**Comment by Atharv-Pathak-14** [comment]
Ok sure you can 👍🏻

**Comment by Dhananjay-JSR** [comment]
https://github.com/railwayapp/devicons/pull/33 

Made Mentioned Changes and requested PR

**Comment by Atharv-Pathak-14** [comment]
I will make a new pr today

**Comment by Dhananjay-JSR** [comment]
No Need there is already a PR in progress linked with this 


https://github.com/railwayapp/devicons/pull/33

**Pull request closed without merging**